### PR TITLE
Objects Automated Test - LPS-141652 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -327,13 +327,92 @@ definition {
...
+
+		Click(locator1 = "ObjectAdmin#LAYOUT_LAYOUT_TAB");
+
+		ObjectAdmin.addTabFieldsOnLayout(
+			blockName = "Block 1",
+			columnsNumber = "1",
+			fieldList = "Custom Field,Custom Object",
+			tabName = "Field Tab");
+
+		ObjectAdmin.addTabFieldsOnLayout(tabName = "Field Tab");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:388]: You don't need use this macro for this test

Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:388]: You don't need to add one more Field Tab on this test

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro [hunk 2]
```diff
@@ -185,6 +185,12 @@ definition {
 		Click(locator1 = "ObjectPortlet#CUSTOM_OBJECT_ON_CONTROL_PANEL");
 	}
 
+	macro gotosRelationshipTab {
+		Click(locator1 = "//a[contains(text(), 'Relationship Tab')]");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:189]: You can use the `ObjectAdmin#ENTRY_RELATIONSHIP_TAB` locator for this macro

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @PauloVAS,

Please, check my comments and make the necessary changes.

Let me know if you have any questions.

Thanks.

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**